src/network/networkd-manager-varlink.c

```diff
@@ -201,7 +201,11 @@ static int vl_method_set_persistent_storage(Varlink *vlink, JsonVariant *paramet
                 }
 
                 if (fstat(fd, &st) < 0)
-                        return log_warning_errno(r, "Failed to stat the persistent storage: %m");
```

Comment:
Maybe just say "Failed to stat the passed persistent storage fd"? "passed file descriptor" might be too vague.